app/src/main/java/com/android/ootd/ui/post/AddItemsScreen.kt

```diff
@@ -212,14 +214,13 @@ fun AddItemsScreen(
   }
 }
 
-// --- smaller composables ---
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 private fun AddItemTopBar(modifier: Modifier, goBack: () -> Unit) {
   CenterAlignedTopAppBar(
       title = {
         Text(
-            text = "ADD ITEMS",
+            text = "ADD ITEM", // ← updated
```

Comment:
Maybe just remove this comment

```suggestion
            text = "ADD ITEM",
```